core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -269,13 +269,19 @@ trait AttributeComponent {
       }
     }
 
-    def findUniqueStringsByNameQuery(attrName: AttributeName, queryString: String) = {
-      filter(rec =>
+    def findUniqueStringsByNameQuery(attrName: AttributeName, queryString: Option[String]) = {
+      // ToDo: figure out what the generated query is
+
+      val basicFilter = filter(rec =>
         rec.namespace === attrName.namespace &&
-        rec.name === attrName.name &&
-        rec.valueString.isDefined &&
-        rec.valueString.like(s"%$queryString%")
-      ).map(_.valueString).distinct
+          rec.name === attrName.name &&
+          rec.valueString.isDefined)
+
+      (queryString match {
+        case Some(query) => basicFilter.filter(_.valueString.like(s"%${query}%"))
```

Comment:
yeah the like means we are case-insensitive, which is what Moran wanted so that we don't have a `Cancer` tag and a `cancer` tag